db/migrate/20171204215532_add_status_to_preserved_copies.rb

```diff
@@ -1,6 +1,11 @@
 class AddStatusToPreservedCopies < ActiveRecord::Migration[5.1]
-  def change
-    add_column :preserved_copies, :status, :int, null: false
+  def up
+    add_column :preserved_copies, :status, :int
+    change_column_null :preserved_copies, :status, false
```

Comment:
so this adds the constraint which tells the database not to accept NULL values?

Reply:
Yes. I just tested this out via console in test with:
```
pc = PreservedCopy.first
pc.status = nil
```
and was not allowed to save:
```
[8] pry(main)> pc.save
   (0.3ms)  BEGIN
  PreservedObject Load (0.3ms)  SELECT  "preserved_objects".* FROM "preserved_objects" WHERE "preserved_objects"."id" = $1 LIMIT $2  [["id", 197], ["LIMIT", 1]]
  Endpoint Load (0.3ms)  SELECT  "endpoints".* FROM "endpoints" WHERE "endpoints"."id" = $1 LIMIT $2  [["id", 1], ["LIMIT", 1]]
   (0.2ms)  ROLLBACK
=> false
```